packages/react-flow-editor/src/Editor/helpers/getDOMRect.ts

```diff
@@ -0,0 +1,7 @@
+export const getDOMRect = (element: Element | null): DOMRect => {
+  const rect = element?.getBoundingClientRect()
+
+  if (!rect) return new DOMRect()
```

Comment:
Is it singleton? Maybe we use this with instance inside DOMRect

Reply:
domrect is native js object, it is need here to create zero-size plug